### ISSUE TITLE
Empty page.collection from modular page

### ISSUE THREAD

**Issue by peachp** [getgrav/grav#2738]
I have a _module (modular page) with following frontmatter:
```
title: 'List subpages'
content:
    items: '@page.children': '/somepage'
```

The Twig template for this module is:
```
{% for p in page.collection %}
<h2>{{ p.title }}</h2>
{% endfor %}
```
Nothing is shown. When I remove the `content` in the frontmatter above and hardcode it in Twig like this:
```
{% set options = { items: {'@page.children': '/somepage'}, 'limit': 10, 'order': {'by': 'date', 'dir': 'desc'}} %}
{% set my_collection = page.collection(options) %}
<ul>
{% for p in my_collection %}
    <li>{{ p.title }}</li>
{% endfor %}
</ul>
```
It works.
Note: what is strange, the "hardcoded" Twig `page.collection(options)` does **not** work until you remove / comment out the `content` in the frontmatter of this module.
I mean OK, fair enough, maybe the `content` is not supported modules, but why would it stop the direct Twig code `page.collection(options)` from working?

Not sure if this is a bug. Maybe the thinking behind it is that yes, getting a collection within a modular template only works in code (for whatever reasons) and there should be no such settings in the frontmatter either, to keep it clean.

---
EDIT:
OK, it looks like there is no need to remove ``content`` and the Twig `page.collection(options)` will still work (was probably caching...). But I still wonder why I can't use the `page.collection` defined in the frontmatter of modular pages.

P.S.: why are modular pages not called "modules"? I remember when I started with Grav, it took me a while to distinguish between the parent / primary modular page and the _modularPage (what I call modules).

**Comment by rhukster** [comment]
I think your problem is this syntax:

```
content:
    items: '@page.children': '/somepage'
```

It should be:

```
content:
    items: 
        '@page.children': '/somepage'
```

or better yet:

```
content:
    items: 
        page@.children:  /somepage
```

if you put the `@` at the end (both are supported), you don't need the single quotes.

Regarding your P.S.  The naming of modular pages have always been a bit clumsy.  This has let to confusion.  We have decided to change the naming to the parent page being 'modular' but the pages that make up a modular page are 'module' pages.  We've not fully updated the code to match this, but we're working on it for Grav 1.7.. we've already changed code references to `modular()` to be `isModule()` instead (although `modular()` is still there for compatibility)
